### PR TITLE
925 - Add responsive design on Monthview (Adds legend and events)

### PR DESCRIPTION
**Pull request by ericangeles**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**
Added the legend and calendar events on mobile view calendar-monthview (Responsive Design).

**Related github/jira issue (required)**:
Closes https://github.com/infor-design/enterprise/issues/925

**Steps necessary to review your pull request (required)**:
- Pull this branch, build, and run the app
- Navigate to http://localhost:4000/components/calendar/test-specific-month.html
- Resize the browser until you reach 967px
- First, you'll see the legend will appear down at the bottom
- Then click on any of event in the calendar
- After clicking the event, a listview component will appear with the information about the event at the bottom, there could be several depending whats on the day
- Test this to different themes and variants
- click the listview and a popup thats full screen should open. At this break point all popups are full screen

**Included in this Pull Request**:
- [x] A note to the change log.

### CODE REVIEW

**Comment by ericangeles** [comment]
This is my initial fix and I suspect there's a lot of bugs, changes or other way to fix this so need your inputs @tmcconechy @EdwardCoyle @deep7102

File: app/views/components/calendar/test-specific-month.html [hunk 1]
```diff
@@ -49,6 +51,7 @@
         events: events,
         upcomingEventDays: 0,
         template: 'tmpl-readonly',
+        mobileTemplate: 'tmpl-mbl-readonly',
```
Comment (tmcconechy) [inline app/views/components/calendar/test-specific-month.html]: Just thinking out loud here as this isn't a bad idea.... Was thinking the same template would just adjust?
Was that not possible? Just a comment for now as this is maybe a better idea as its more flexible.

Or maybe add the new markup for the legend to the other template and just hide/show?

**Comment by ericangeles** [comment]
Making it WIP at this moment as I am implementing another legends for events and trying to make the template more reusable instead of creating another one.

File: app/views/components/calendar/test-specific-month.html (hunk 1 above)
Comment (ericangeles) [reply app/views/components/calendar/test-specific-month.html]: All noted, @tmcconechy. Thank you.

**Comment by ericangeles** [comment]
@tmcconechy do we have a sample where the listview has a drill? Or this will be a customization also?

**Comment by ericangeles** [comment]
I also reuse the template, seems to work but having multiple events on mobile doesn't loop inside of the `<li>` of listview container, instead, it creates another one (listview container). 

See this video link
https://drive.google.com/file/d/1fKD344b4NsZk9pr4M38YmSU698jAP8uT/view

**Comment by tmcconechy** [comment]
@ericangeles we have the behavior here https://master-enterprise.demo.design.infor.com/patterns/list-detail.html if you go mobile. But it doesnt have the > icon. We have the icon so i think just need to add that and see about reusing that behavior.

**Comment by tmcconechy** [comment]
Still working on this one too? Or does it need help?

**Comment by ericangeles** [comment]
@tmcconechy Need help on this.

**Comment by tmcconechy** [comment]
Working on this now @ericangeles - figured out some things but still working

**Comment by tmcconechy** [comment]
@ericangeles can you review (your own PR :)) I improved this quite a bit now. Think its good enough for now between the two of us

Everyone else this is ready.

**Pull request merged**